### DISCUSSION TITLE
problem trying to add cc tweaked compatibility on multiloader

### DISCUSSION THREAD

**Discussion by Szedann** [cc-tweaked/CC-Tweaked#1801] [Q&A] [answered]
when trying to load into a world, the internal server crashes with this report.
the crash only happens on forge.
[crash-2024-04-20_18.53.54-server.txt](https://github.com/cc-tweaked/CC-Tweaked/files/15052395/crash-2024-04-20_18.53.54-server.txt)

project: https://github.com/Layers-of-Railways/CreateNumismatics/tree/1.20.1/computercraft

cc dependency in common:
https://github.com/Layers-of-Railways/CreateNumismatics/blob/19a909fef664c52a8cba2c369297417b9f2a32d9/common/build.gradle.kts#L55

cc dependency in forge:
https://github.com/Layers-of-Railways/CreateNumismatics/blob/19a909fef664c52a8cba2c369297417b9f2a32d9/forge/build.gradle.kts#L47-L51

file crashing:
https://github.com/Layers-of-Railways/CreateNumismatics/blob/19a909fef664c52a8cba2c369297417b9f2a32d9/common/src/main/java/dev/ithundxr/createnumismatics/compat/computercraft/implementation/peripherals/BrassDepositorPeripheral.java

**Comment by SquidDev** [comment, marked as answer]
This looks to be the same issue as #1752. I'm afraid this is caused by a bug in Architectury loom (https://github.com/architectury/architectury-loom/pull/159), but you can work around by including the dependencies explicitly.
